shell/browser/api/electron_api_session.cc

```diff
@@ -455,6 +487,19 @@ v8::Local<v8::Promise> Session::SetProxy(gin::Arguments* args) {
   return handle;
 }
 
+v8::Local<v8::Promise> Session::ForceReloadProxyConfig() {
+  auto* isolate = v8::Isolate::GetCurrent();
```

Comment:
please use `auto* isolate = JavascriptEnvironment::GetIsolate();` instead

Reply:
Ok, got it.